test-apps/ui-test-app/src/frontend/index.tsx

```diff
@@ -334,7 +334,6 @@ export class SampleAppIModelApp {
 
     await FrontendDevTools.initialize();
     await HyperModeling.initialize();
-    await MapLayersUI.initialize({ iTwinConfig: new MichelTestPrefs(), featureInfoOpts: { onMapHit: DefaultMapFeatureInfoTool.onMapHit } });

```

Comment:
@mdastous-bentley Should the full MapLayersUI be removed, or only the iTwinConfig ?